### PR TITLE
Add support for reporting metrics to Librato

### PR DESCRIPTION
**Pull request by lifeeth**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -85,4 +95,14 @@ VOLUME ["/sys/fs/cgroup"]
 VOLUME ["/run"]
 VOLUME ["/run/lock"]
 
+# Librato default configuration
+COPY librato/df.conf /opt/collectd/etc/collectd.conf.d/df.conf
```
Comment (Page-) [inline Dockerfile]: Can you merge these into one `COPY` please

File: Dockerfile [hunk 2]
```diff
@@ -85,4 +95,14 @@ VOLUME ["/sys/fs/cgroup"]
...
+COPY librato/disk.conf /opt/collectd/etc/collectd.conf.d/disk.conf
+COPY librato/interface.conf /opt/collectd/etc/collectd.conf.d/interface.conf
+COPY librato/collectd.service /lib/systemd/system/collectd.service
+COPY librato/source_name.sh /opt/collectd/bin/source_name.sh
+ADD librato/confd /opt/collectd/etc/confd
```
Comment (Page-) [inline Dockerfile]: Do we need `ADD` specifically?

Comment (lifeeth) [inline Dockerfile]: Could be a COPY too - Switching it.

File: Dockerfile (hunk 1 above)
Comment (lifeeth) [inline Dockerfile]: Done. I wanted the changes to be explicit and so had them separately.

File: librato/confd/conf.d/librato.toml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[template]
+src = "librato.conf.tmpl"
+dest = "/opt/collectd/etc/collectd.conf.d/librato.conf"
+keys = [
+  "/resin/librato/user",
```
Comment (petrosagg) [inline librato/confd/conf.d/librato.toml]: Can we namepsace them under `common`?

Comment (lifeeth) [inline librato/confd/conf.d/librato.toml]: Yup - will do that.

Comment (lifeeth) [inline librato/confd/conf.d/librato.toml]: Done.

**Comment by petrosagg** [comment]
Your PR includes a commit that is already merged. in general SGTM

**Comment by lifeeth** [comment]
Just cleaned that up!

File: Dockerfile [hunk 4]
```diff
@@ -85,4 +95,12 @@ VOLUME ["/sys/fs/cgroup"]
 VOLUME ["/run"]
 VOLUME ["/run/lock"]
 
+# Librato default configuration
+COPY librato/*.conf /opt/collectd/etc/collectd.conf.d/
+COPY librato/collectd.service /lib/systemd/system/collectd.service
+COPY librato/source_name.sh /opt/collectd/bin/source_name.sh
+COPY librato/confd /opt/collectd/etc/confd
+
+RUN rm /opt/collectd/etc/collectd.conf.d/swap.conf && rm /opt/collectd/etc/collectd.conf.d/librato.conf
```
Comment (Page-) [inline Dockerfile]: Should this be above with the `&& rm collectd*.deb \` stuff?

Comment (lifeeth) [inline Dockerfile]: Nope one is installation - other is managing the config files.

**Comment by Page-** [comment]
test this please

**Comment by lifeeth** [comment]
Decided to drop librato collection.

**Pull request closed without merging**